Kajabi/sage-archived
# Sage icon typo fix

## Description
There is a typo for the `favorite` icon that needs to be fixed.

### Steps for testing
1. Visit http://localhost:4000/pages/foundations/icon
1. Ensure that the `favorite` icon is spelled correctly
